dotcom-rendering/package.json

```diff
@@ -96,7 +96,7 @@
     "@types/lodash.debounce": "^4.0.6",
     "@types/lodash.escape": "^4.0.6",
     "@types/lodash.get": "^4.4.4",
-    "@types/node": "^14.14.24",
+    "@types/node": "^17.0.18",
```

Comment:
the `nvmrc` still mandates 14: https://github.com/guardian/dotcom-rendering/blob/d20de81f93aee130cf425889f456ab798b9a5aed/.nvmrc#L1

did you mean this?